MAINTAINERS.yml

```diff
@@ -2346,18 +2346,25 @@ Nuvoton NPCX Platforms:
   labels:
     - "platform: Nuvoton NPCX"
 
-Nuvoton Numicro Platforms:
-  status: odd fixes
+Nuvoton Numicro Numaker Platforms:
+  status: maintained
+  maintainers:
+    - cyliangtw
   collaborators:
     - ssekar15
+    - fabiobaltieri
```

Comment:
I'm humbled and honored but I think you are better off with just yourself and @ssekar15 on the list, feel free to add me as reviewer to PRs if you need but I think for now I'm happy to stay off the list for platforms I don't actively work on myself. :-)

Reply:
Sorry for the inconvenience, I've removed you from the list.

Reply:
Nothing to be sorry about, thanks for doing the change. :-) Always cool to see new platforms maintained!